browser/ui/webui/settings/brave_default_extensions_handler.cc

```diff
@@ -112,6 +157,26 @@ void BraveDefaultExtensionsHandler::OnInstallResult(
   }
 }
 
+void BraveDefaultExtensionsHandler::SetMediaRouterEnabled(
+    const base::ListValue* args) {
+  CHECK_EQ(args->GetSize(), 1U);
+  CHECK(profile_);
+  bool enabled;
+  args->GetBoolean(0, &enabled);
+
+  restartNeeded = !restartNeeded;
+  if (enabled) {
```

Comment:
It would be good to add some comments why this is needed.

Reply:
done!